src/backend/commands/player/troll/window_tint.cpp

```diff
@@ -0,0 +1,27 @@
+#include "backend/player_command.hpp"
+#include "natives.hpp"
+#include "pointers.hpp"
+#include "util/teleport.hpp"
+#include "util/vehicle.hpp"
+
+namespace big
+{
+	class black_tint : player_command
+	{
+		using player_command::player_command;
+
+		virtual void execute(player_ptr player, const std::vector<std::uint64_t>& _args, const std::shared_ptr<command_context> ctx)
+		{
+		    Ped ped = PLAYER::GET_PLAYER_PED_SCRIPT_INDEX(player->id());
+		    if (PED::IS_PED_IN_ANY_VEHICLE(ped, false))
+		    {
+		    	entity::take_control_of(PED::GET_VEHICLE_PED_IS_USING(ped));
+		    	Vehicle UserVeh = PED::GET_VEHICLE_PED_IS_IN(ped, false);
+
+		    	VEHICLE::SET_VEHICLE_WINDOW_TINT(UserVeh, WINDOWTINT_BLACK);
+		    }
+        }
+	};
```

Comment:
Weird indentation

Reply:
I hate this indentation shit

Reply:
![image](https://user-images.githubusercontent.com/121067410/214407875-a9c9df80-b966-4352-9f7b-8f96c278a1dc.png)
dont look like tha for me

Reply:
> dont look like tha for me

It will not be displayed in the editor, just replace all TABs with four spaces or vice versa